app/views/proctor/registrations/show.html.erb

```diff
@@ -7,14 +7,14 @@
       <% if @registration.anomalous? %>
         <p><%= link_to "Show anomalies", exam_registration_anomalies_path(@registration.exam, @registration), class: "btn btn-danger" %></p>
       <% end %>
-      <p><%= link_to "View exam registrations", exam_registrations_path(@registration.exam), class: "btn btn-info" %></p>
-      <p><%= link_to "View exam", @registration.exam, class: "btn btn-info" %></p>
-      <p><%= link_to "View room", [@exam, @registration.room], class: "btn btn-info" %></p>
+      <p><%= link_to "View exam registrations", proctor_exam_registrations_path(@registration.exam), class: "btn btn-info" %></p>
+      <p><%= link_to "View exam", ['proctor', @registration.exam], class: "btn btn-info" %></p>
+      <p><%= link_to "View room", ['proctor', @exam, @registration.room], class: "btn btn-info" %></p>
```

Comment:
It's weird to me that these three `link_to`'s aren't using the same syntax; chalk it up to Rails, I guess?